astronomer/starship/templates/starship/dags.html

```diff
@@ -11,12 +11,13 @@
             <th scope="col">Present on Astro?</th>
             <th scope="col" style="width: 8%">Local</th>
             <th scope="col" style="width: 8%">Remote</th>
+            <th scope="col" style="width: 8%">Migrate</th>
         </tr>
         </thead>
         <tbody>
         {% for k, v in data.dags.items() %}
         <tr id="dag-{{k}}"
-            hx-get="{{ url_for('AstroMigration.dag_cutover_row_get', deployment=session.selectedAstroDeployment or 'undefined', dag_id=k) }}"
+            hx-get="{{ url_for('AstroMigration.dag_cutover_row_get', deployment=session.selectedAstroDeployment or 'undefined', dag_id=k, action='init') }}"
```

Comment:
I added this - but I'm not actually sure it was needed? `action='init'` and adding that to the "must be one of these actions" list? I wasn't sure what the page should render on a `GET` - I guess it just doesn't do any of that stuff?